public/locales/pt/affix.json

```diff
@@ -1,8 +1,8 @@
 {
-    "affixButtonTitle": "Clique aqui para adicionar um discurso ou personalidade",
-    "affixButtonCreateClaim": "Clique aqui para adicionar um discurso",
+    "affixButtonTitle": "Clique aqui para adicionar uma afirmação ou personalidade",
+    "affixButtonCreateClaim": "Clique aqui para adicionar uma afirmação",
     "affixButtonCreatePersonality": "Clique aqui para adicionar uma personalidade",
-    "affixButtonCreateVerifiedSources": "Clique aqui para adicionar uma informação checada",
+    "affixButtonCreateVerifiedSources": "Clique aqui para adicionar uma fonte",
```

Comment:
I don't think this is the best wording. Verified Sources is kind of a fact-check for source reliability.